python/pyarrow/tests/test_compute.py

```diff
@@ -1994,12 +2000,12 @@ def test_assume_timezone():
                                  "2009-12-31T04:20:20.004132",
                                  "2010-01-01T05:25:25.005321",
                                  "2010-01-03T06:30:30.006163",
-                                 "2010-01-04T07:35:35",
-                                 "2006-01-01T08:40:40",
-                                 "2005-12-31T09:45:45",
-                                 "2008-12-28",
-                                 "2008-12-29",
-                                 "2012-01-01 01:02:03"])
+                                 "2010-01-04T07:35:35.0",
```

Comment:
just curious why are those being updated? are formats like `"2008-12-28"` not supported anymore?

Reply:
That string itself, yes, but not the mixture of different formats in a single list (but I am planning to raise an issue about this, because it seems too pedantic, as all those strings are ISO strings)